src/c2_blockchain/p4_batched_extrinsics.rs

```diff
@@ -152,25 +154,23 @@ fn part_4_child_block() {
 }
 
 #[test]
-fn part_4_child_header() {
+fn bc_4_child_header() {
     let g = Header::genesis();
-    let h1 = g.child(hash(&vec![1, 2, 3]), 6);
+    let h1 = g.child(hash(&[1, 2, 3]), 6);
 
-    assert_eq!(h1.height, 1);
-    assert_eq!(h1.parent, hash(&g));
-    assert_eq!(h1.extrinsics_root, hash(&vec![1, 2, 3]));
+    assert!(g.verify_child(&h1));
```

Comment:
We shouldn't use methods like `verify_child` here because the students will have written those methods.

It may be the case that their code to generate a header is wrong and their code to check a header is also wrong in the same way. So we need to actually check the parent hash and extrinsics root individually.

Reply:
Yes of course stupid